implementations.md

```diff
@@ -0,0 +1,13 @@
+# Implementations of Container Specification
```

Comment:
Maybe just “Implementations” here?  The fact that we're talking about this specification should be clear from the context.